### ISSUE TITLE
We need an "Turn off PC" when finish a download, or all downloads

### ISSUE THREAD

**Issue by jesquivelm** [transmission/transmission#7084]
### What is the issue?

We need an "Turn off PC" when finish a download, or all downloads. If there is an option please let us know how to configure or how do we should do the script on different OS. 

### Which application of Transmission?

Qt app on Windows

**Comment by ILoveGoulash** [comment]
Should probably be doable with the completed script and an RPC call to get the number of in-progress downloads remaining.

Please don't ask of a torrent client to make coffee and tuck you in your bed without looking the various IPC methods first.

**Comment by zwimer** [comment]
This seems like it would discourage seeding what was downloaded.

**Comment by Tharn** [comment]
Yep, this sounds very much like "you", not "we". Transmission does not always shut down gracefully, reporting stats fully to all trackers etc. And it happens with other clients too. What you're asking is to be given a tool to be a _bad citizen._

**Comment by ckerr** [comment]
This is a pretty niche use case so I don't think this is going to be supported in the UI.

Agree with @ILoveGoulash that people who need this functionality could do it in a torrent-done script

**Comment by ckerr** [comment]
Also, duplicate of #396
